integration/nwo/cmd/cryptogen/cryptogen.go

```diff
@@ -230,129 +228,6 @@ func getConfig() (*Config, error) {
 	return config, nil
 }
 
-func extend() {
-	config, err := getConfig()
-	if err != nil {
-		fmt.Printf("Error reading config: %s", err)
-		os.Exit(-1)
-	}
-
-	for _, orgSpec := range config.PeerOrgs {
-		err = renderOrgSpec(&orgSpec, "peer")
-		if err != nil {
-			fmt.Printf("Error processing peer configuration: %s", err)
-			os.Exit(-1)
-		}
-		extendPeerOrg(orgSpec)
-	}
-
-	for _, orgSpec := range config.OrdererOrgs {
-		err = renderOrgSpec(&orgSpec, "orderer")
-		if err != nil {
-			fmt.Printf("Error processing orderer configuration: %s", err)
-			os.Exit(-1)
-		}
-		extendOrdererOrg(orgSpec)
-	}
-
-}
-
-func extendPeerOrg(orgSpec OrgSpec) {
```

Comment:
@adecaro Is this something we still need?

Reply:
we never used it, actually. We can always bring it back if we decide to enhance our NWO.